ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -198,6 +198,9 @@ public class BottomSheetController: UIViewController {
         }
     }
 
+    /// When enabled, users will be able to move the sheet to the hidden state by swiping down.
+    @objc open var allowsSwipeToHide: Bool = false
```

Comment:
should we be explicit this isn't a thing in iPad?

Reply:
It works the same way on iPad when it's enabled.

Reply:
i thought bottomsheet was a floating bar in ipad?

Reply:
No only bottom commanding behaves that way. Bottom sheets work on all devices.